src/eventHandlers/continuousIntegrationEnd/majorBump.spec.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @webhook-pragma check_suite
+ */
+
+import * as core from '@actions/core';
+import { getOctokit } from '@actions/github';
+import { OK } from 'http-status-codes';
+import * as nock from 'nock';
+
+import { continuousIntegrationEndHandle } from '.';
+
+/* cspell:disable-next-line */
+const PULL_REQUEST_ID = 'MDExOlB1bGxSZXF1ZXN0MzE3MDI5MjU4';
+const COMMIT_HEADLINE = 'Update test';
+
+const octokit = getOctokit('SECRET_GITHUB_TOKEN');
+const getInputSpy = jest.spyOn(core, 'getInput').mockImplementation();
+
+jest.spyOn(core, 'info').mockImplementation();
+
+beforeEach((): void => {
+  getInputSpy.mockReturnValue('DEPENDABOT_PATCH');
+});
+
+describe('continuous integration end event handler', (): void => {
```

Comment:
What is the difference between this test and the one [here](https://github.com/ridedott/merge-me-action/blob/master/src/eventHandlers/checkSuite/index.spec.ts#L458)

Reply:
You're right, it's the same, I will remove it :-)